src/pages/login.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { ReactElement } from 'react';
+import { NextPageContext } from 'next';
+import { parseCookies } from 'nookies';
+import Layout from '../layout/Layout';
```

Comment:
Since the page has two columns, we need to import BaseLayout instead and use that, `import { BaseLayout } from '../layout/Layout'`

Reply:
Now importing and using BaseLayout.